src/plugins/index.js

```diff
@@ -106,6 +107,12 @@ export function getActivityTrackingPlugin({ agent, isProd }) {
 	};
 }
 
+export function getClickTrackingPlugin() {
```

Comment:
just curious why this little preamble is here. i see that other plugins have it as well but it remains mysterious.